src/sentry/web/frontend/auth_login.py

```diff
@@ -432,7 +432,7 @@ def get_ratelimited_login_form(
         ]
         metrics.incr("login.attempt", instance="rate_limited", skip_internal=True, sample_rate=1.0)
 
-        context = {
+        context = self.get_default_context(request=request) | {
```

Comment:
I'm assuming we won't run into any python incompatiblity issues since this is 3.9+?

Reply:
Yup <3